### PR TITLE
Moving msbuild dependencies to msbuild 15.

### PR DESCRIPTION
**Pull request by livarcocc**
@eerhardt @brthor @Sridhar-MS @piotrpMSFT @jeffkl

### CODE REVIEW

File: src/redist/project.json [hunk 1]
```diff
@@ -19,12 +19,11 @@
     "tool_nuget": "1.0.0-preview3-*",
     "tool_msbuild": "1.0.0-preview3-*",
 
-    "MSBuild": "0.1.0-preview-00043-160929",
-    "Microsoft.Build.Framework": "0.1.0-preview-00043-160929",
-    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00043-160929",
-    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00043-160929",
-    "Microsoft.Build.Targets": "0.1.0-preview-00043-160929",
-    "Microsoft.Build": "0.1.0-preview-00043-160929",
+    "Microsoft.Build.Runtime": "15.1.298-preview5",
```
Comment (eerhardt) [inline src/redist/project.json]: Don't you just need `Microsoft.Build.Runtime` and the rest come with it?

Comment (jeffkl) [inline src/redist/project.json]: Yes I would recommend if you don't have any compile-time references that you only add `Microsoft.Build.Runtime`.  It's dependencies include our other binaries.

**Comment by eerhardt** [comment]
@livarcocc - an option to get unblocked for your restore problem is to add `/p:SkipInvalidConfigurations=true` to your call to `dotnet restore3`.  This will disable the check that is raising an error.

**Comment by eerhardt** [comment]
:shipit:

**Pull request merged**